src/node_api.cc

```diff
@@ -1427,3 +1427,34 @@ napi_status NAPI_CDECL node_api_get_module_file_name(
   *result = static_cast<node_napi_env>(env)->GetFilename();
   return napi_clear_last_error(env);
 }
+
+napi_status NAPI_CDECL
```

Comment:
The function implementation must be wrapped into `#ifdef NAPI_EXPERIMENTAL`

Reply:
thanks ı applied